content/people/michaeltunnell.md

```diff
@@ -1,13 +1,12 @@
 ---
 avatar: /images/people/michaeltunnell.jpg
-avatar_small: /images/people/michaeltunnell_small.jpg
-bio: Podcaster / Content Creator for the Destination Linux Podcast. I'm also a Designer,
-  Marketer, Writer, Open Sourcerer, Linux Enthusiast, and host of This Week in Linux.
+avatar_small: null
```

Comment:
Are these needed/used by the website guest page? We can probably cleanup the entry some, just want to make sure we don't make the page look worse as a result.

Probs at least partially my fault, added a podcast:person entry for him in a recent ep, but it was quite minimal compared to some of our other info. Eventually we'll get this better unified.

Reply:
and thank you for looking into this! :pray:

Reply:
avatar_small is not currently used in the site and is limited to Fireside sites, and also not supported by [podcast:person](https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#person). We could add something to the scraper to make a smaller thumbnail image but I don't know that it is needed.